_posts/2021-03-17-#21142.md

```diff
@@ -10,13 +10,51 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Fuzzing is an effective way to find vulnerabilities in code before it is
+  merged or released. A [wiki
+  page](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/Fuzz-Trophies)
+  collects the issues fuzzing could identify.
+
+- Bitcoin Core has [over
+  160](https://github.com/bitcoin/bitcoin/tree/master/src/test/fuzz) individual
+  fuzz targets, but the code coverage of consensus, validation and P2P code
+  paths is [still
+  low](https://marcofalke.github.io/btc_cov/fuzz.coverage/index.html). For
+  example `MemPoolAccept::AcceptSingleTransaction` never gets past `PreChecks`
+  and exits early.
+
+- `MemPoolAccept` is used to accept (or reject) transaction from remote peers, the wallet and the RPC interface.
+
+- The existing `process_message` fuzz target can also test messages of type
+  `tx`. It works by [initializing a
+  blockchain](https://github.com/bitcoin/bitcoin/blob/63314b8211d795b2bf0814e244d801e74f50b152/src/test/fuzz/process_message.cpp#L63),
+  then it lets the fuzz engine create a message (e.g. `tx`) from a peer [to be
+  processed](https://github.com/bitcoin/bitcoin/blob/63314b8211d795b2bf0814e244d801e74f50b152/src/test/fuzz/process_message.cpp#L95).
+
+- The new `tx_pool` fuzz target aims to fuzz mempool acceptance more efficient.
```

Comment:
```suggestion
- The new `tx_pool` fuzz target aims to make fuzz mempool acceptance more efficient.
```

Reply:
I think this is right `fuzz` is the verb, not a noun

Reply:
Ah, then s/efficient/efficiently/

Reply:
thanks, done